### DISCUSSION TITLE
Is there a functional reason for SOFA to read ".lab" transcriptions instead of ".txt"?

### DISCUSSION THREAD

**Discussion by spicytigermeat** [qiuqiao/SOFA#10] [Q&A] [answered]
I wonder if it would be worth changing, as most users won't use .lab for transcriptions, just .txt. Other forced aligners function similarly using .txt.

**Comment by qiuqiao** [comment, marked as answer]
For Chinese users, .lab files are the preferred format for transcriptions instead of .txt files. This is because the Montreal Forced Aligner (MFA) utilizes the .lab format, and consequently, nearly all tools in the DiffSinger dataset ecosystem, including LyricFA and minlabel, are designed to work with the .lab extension.

**Reply by spicytigermeat** [reply]
Okay, that makes sense! Though, I've always used ".txt" with MFA. Maybe that can be a choice when inferring?

**Reply by qiuqiao** [reply]
I've committed this feature, but it hasn't been tested. Could you help me test it?

**Reply by spicytigermeat** [reply]
Yes! I just saw the commit, I'll give it a test right now :)

**Reply by spicytigermeat** [reply]
There was one small error but I fixed it and submitted a pull request :)

**Reply by qiuqiao** [reply]
Thank you very much.
